var/spack/repos/builtin/packages/py-google-crc32c/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGoogleCrc32c(PythonPackage):
+    """This package wraps the google/crc32c hardware-based implementation
+    of the CRC32C hashing algorithm."""
+
+    homepage = "https://github.com/googleapis/python-crc32c"
+    git      = "https://github.com/googleapis/python-crc32c"
```

Comment:
It's better to download directly from PyPI so we can checksum:
```suggestion
   pypi = "google-crc32c/google-crc32c-1.3.0.tar.gz"
```
See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Reply:
Ah, I seem to have overlooked the tar.gz file on pypi. Fixed it now.